examples/guided.py

```diff
@@ -205,21 +269,35 @@ def ask_user_questions():
 		archinstall.arguments['kernels'] = archinstall.select_kernel(kernels)
 
 	# Additional packages (with some light weight error handling for invalid package names)
-	print("Only packages such as base, base-devel, linux, linux-firmware, efibootmgr and optional profile packages are installed.")
-	print("If you desire a web browser, such as firefox or chromium, you may specify it in the following prompt.")
+	print(
+		"Only packages such as base, base-devel, linux, linux-firmware, efibootmgr and optional profile packages are installed."
+	)
+	print(
+		"If you desire a web browser, such as firefox or chromium, you may specify it in the following prompt."
+	)
 	while True:
 		if not archinstall.arguments.get('packages', None):
-			archinstall.arguments['packages'] = [package for package in input('Write additional packages to install (space separated, leave blank to skip): ').split(' ') if len(package)]
+			archinstall.arguments['packages'] = [
+				package
+				for package in input(
+					'Write additional packages to install (space separated, leave blank to skip): '
+				).split(' ')
+				if len(package)
+			]
 
 		if len(archinstall.arguments['packages']):
 			# Verify packages that were given
 			try:
-				archinstall.log(f"Verifying that additional packages exist (this might take a few seconds)")
+				archinstall.log(
+					f"Verifying that additional packages exist (this might take a few seconds)"
+				)
 				archinstall.validate_package_list(archinstall.arguments['packages'])
 				break
 			except archinstall.RequirementError as e:
 				archinstall.log(e, fg='red')
-				archinstall.arguments['packages'] = None # Clear the packages to trigger a new input question
+				archinstall.arguments[
+					'packages'
+				] = None  # Clear the packages to trigger a new input question
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability. The fuck is the linter doing..